src/shared/components/RadioButton/RadioButton.tsx

```diff
@@ -1,24 +1,33 @@
 import React from 'react'
-import { Label, Input, StyledInput, StyledLabelText, RadioButtonStyleProps } from './RadioButton.style'
+import {
+  Label,
+  Input,
+  StyledInput,
+  StyledLabelText,
+  StyledHelperText,
+  RadioButtonStyleProps,
+} from './RadioButton.style'
 
 type RadioButtonProps = Partial<{
   selectedValue: string | number
   label: string
+  helperText?: string
 }> &
   Omit<RadioButtonStyleProps, 'clickable'> &
   React.DetailedHTMLProps<React.InputHTMLAttributes<HTMLInputElement>, HTMLInputElement> &
   Omit<React.InputHTMLAttributes<HTMLInputElement>, 'onClick'>
 
 const RadioButton = React.forwardRef<HTMLInputElement, RadioButtonProps>(
-  ({ value, selectedValue, label, disabled, error, ...props }, ref) => {
+  ({ value, selectedValue, label, helperText, disabled, error, ...props }, ref) => {
     const isSelected = value === selectedValue
 
     return (
       <Label disabled={disabled}>
         <StyledInput checked={isSelected} error={error} disabled={disabled}>
```

Comment:
Spacing here needs to be improved, like mentioned in the chat:

<img width="958" alt="Screenshot 2021-04-09 at 11 08 43" src="https://user-images.githubusercontent.com/12646744/114158254-c8b35380-9924-11eb-9447-e3a801c87168.png">
<img width="502" alt="Screenshot 2021-04-09 at 11 12 01" src="https://user-images.githubusercontent.com/12646744/114158264-ca7d1700-9924-11eb-9b52-54af34986057.png">